meson.build

```diff
@@ -0,0 +1,79 @@
+project(
+    'minmea',
+    'c',
+    default_options: ['c_std=c99', 'buildtype=debug'],
+)
+
+add_project_arguments(
+    '-Wall', '-Wextra', '-Werror',
+    '-D_POSIX_C_SOURCE=199309L', '-D_BSD_SOURCE', '-D_DEFAULT_SOURCE',
+    '-D_DARWIN_C_SOURCE',
+    language: 'c',
+)
+
+if host_machine.system() == 'windows'
+    add_project_arguments('DMINMEA_INCLUDE_COMPAT')
+endif
+
+c_args = []
+
+if get_option('buildtype') == 'debug'
+    c_args += '-ggdb'
+endif
+
+minmea_includes = include_directories('.')
+
+minmea_sources = ['minmea/minmea.c']
+
+minmea_dependencies = []
+
+minmea_lib = library(
+    'minmea',
+    minmea_sources,
+    include_directories: minmea_includes,
+    dependencies: minmea_dependencies,
+    c_args: c_args,
+)
+
+minmea_dep = declare_dependency(
+    include_directories: minmea_includes,
+    dependencies: minmea_dependencies,
+    link_with: minmea_lib,
+)
+
+if get_option('example')
```

Comment:
I would not bother with this option, just always define the example to be built. It's a single compile rule with no dependencies, it doesn't add any complexity to building, so there's no cost to always building it.

If you just don't want to run an extraneous build rule when using this as a wrap, set this executable to `build_by_default: false`.